Edi/Edi.Apps/Behaviors/ActivatedCommand.cs

```diff
@@ -109,15 +107,13 @@ private static void OnCommandChange(DependencyObject d, DependencyPropertyChange
 		/// <param name="e"></param>
 		private static void UiElement_Activated(object sender, System.EventArgs e)
 		{
-			var uiElement = sender as Window;
```

Comment:
Not OK.
We introduced this style with this commit:
https://github.com/Dirkster99/Edi/commit/e90fd24b2cd5e4cea91553f11719fe9161b46a3e

and I don't really see an improvement here.